src/Tracing/Instrumentation/EventSubscriber/RequestEventSubscriber.php

```diff
@@ -69,14 +69,12 @@ public function __construct(
     public function onRequestEvent(Event\RequestEvent $event): void
     {
         $request = $event->getRequest();
+        $startTime = $request->server->get('REQUEST_TIME_FLOAT'); // Float with microsecond precision
```

Comment:
As before I suggest to retrieve the current time as soon as possible.
And remove the comment since a name has the advantage of providing the information everywhere it's used and not becoming stale.
```suggestion
        $startTimeInMs = $request->server->get('REQUEST_TIME_FLOAT');
        $currentTimeInMs = microtime(true);
```

As for the commands, there is other subscribers/listeners being executing before this one.  
As a reminder `bin/console debug:event-dispatcher kernel.request` list those registered in a SF application with their priority.